tests/unit/test_views.py

```diff
@@ -207,16 +208,30 @@ def test_esi_flash_messages():
 
 class TestSearch:
 
+    def _filter_query(self, s):
+        matches = re.findall(r'(?:"([^"]*)")|([^"]*)', s)
+        result_quoted = [t[0].strip() for t in matches if t[0]]
+        result_unquoted = [t[1].strip() for t in matches if t[1]]
+        return result_quoted, result_unquoted
+
+    def _form_query(self, query_type, query):
+        return Q('multi_match', fields=SEARCH_FIELDS,
```

Comment:
we should combine the fields with boosts here. `multi_match` accepts a list of fields as strings with an optional `^BOOST` suffix, so something like `["normalized_name^10", "description^5"]`

Reply:
I missed this - I'll put it in.